apm-agent-core/src/test/java/co/elastic/apm/agent/configuration/source/ConfigSourcesTest.java

```diff
@@ -39,7 +39,7 @@ void loadFromClasspath() throws IOException {
 
         assertThat(source.getName()).isEqualTo("classpath:test.elasticapm.properties");
         assertThat(source.getValue("application_packages")).isEqualTo("co.elastic.apm");
-        assertThat(source.getValue("log_level")).isEqualTo("DEBUG");
+        assertThat(source.getValue("cloud_provider")).isEqualTo("NONE");
```

Comment:
[for reviewer] this env variable is no longer set in this test config file, however we can test with another.